src/operators.jl

```diff
@@ -509,7 +509,19 @@ sparse(g::AbstractGraph) = adjacency_matrix(g)
 
 length(g::AbstractGraph) = widen(nv(g)) * widen(nv(g))
 ndims(g::AbstractGraph) = 2
-issymmetric(g::AbstractGraph) = !is_directed(g)
+
+@traitfn function issymmetric(g::AG) where {AG <: AbstractGraph; !IsDirected{AG}}
+    return true
+end
+
+@traitfn function issymmetric(g::AG) where {AG <: AbstractGraph; IsDirected{AG}}
+    for e in edges(g)
+        if !has_edge(g, dst(e), src(e))
+            return false
+        end
```

Comment:
Can we change this do something like
```julia
if !has_edge(g, reverse(e))
            return false
        end
```
?
Otherwise it will return incorrect values for graphs with different edge types. An other option would be to not provide a default implementation for directed graphs.